src/net/protocol_agnostic/stream.rs

```diff
@@ -79,20 +94,25 @@ impl PaStream {
         <C as Sink>::SinkError: fmt::Debug,
         C: 'static,
     {
+        let disable_tcp = match config.read().dev {
```

Comment:
Duplicate code: https://github.com/maidsafe/crust/pull/909/commits/e88a2c8990d7bf004c4a7c18c423a11f48a64dc3#diff-2769f48124e6f45bc5d555b5c1d61f29R56
What about moving this to config class, hence smth like `if config.read().tcp_disabled()` could be possible?